java/src/main/c++/vw_learner_VWLearners.cc

```diff
@@ -48,3 +61,74 @@ JNIEXPORT jobject JNICALL Java_vw_learner_VWLearners_getReturnType(JNIEnv *env,
   return env->GetStaticObjectField(clVWReturnType, field);
 }
 
+// Here are all the prediction functions we might want to use.  Add to this as more predictions are required.
+jint multiclass_predictor(example* vec, JNIEnv *env) { return vec->pred.multiclass; }
```

Comment:
@JohnLangford this needs VERY careful review.  Because I've separated the return type from the function that creates it, I now need to create all the prediction functions in one place.  This is that place.  Basically I need to be able to tie the VW arguments with the return function.  Can you please review this and the function `Java_vw_learner_VWLearners_getPredictionFunction` to make sure that all combination of input arguments are appropriately mapped to their output function.